### PR TITLE
Issue #14537 - Update numpy to 1.16

### PR DESCRIPTION
**Pull request by sgfoote**
Change-Id: Ia1494de49dad782d439010b5624b6d18875b86eb

Part of merging QARTOD into baseline.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 geoalchemy2==0.4.0
-numpy==1.12.0
+numpy==1.16
```
Comment (tlfish3638) [inline requirements.txt:2]: Where is this used?

Comment (tlfish3638) [reply requirements.txt:2]: I mean where is this setting in requirements.txt used?

Comment (sgfoote) [reply requirements.txt:2]: The requirements.txt is part of the old school (non-conda) python packaging process. Some people use it with pip to create an environment that can run the code.

**Review by tlfish3638 — APPROVED** [review]
Thanks

**Review by steinermg — APPROVED** [review]

**Pull request merged**